Dockerfile

```diff
@@ -26,4 +26,5 @@ WORKDIR /data/app
 ADD . .
 
 EXPOSE 8080
+HEALTHCHECK CMD curl --fail http://localhost:5000/health-check || exit 1
```

Comment:
It should check proper healthcheck. Normal for API container and celery for celery container

Reply:
Okay that means I have to add for respective containers in docker-compose file. THanks

Reply:
No

Reply:
then two healthcheck command in this DOCKERFILE? like this? 

```
HEALTHCHECK CMD curl --fail http://localhost:5000/health-check || exit 1
HEALTHCHECK CMD curl --fail http://localhost:5000/health-check?q=celery || exit 1
```

Reply:
There can't be 2 HEALTHCHECKs

Reply:
@iamareebjamal Their can be only 1 healthcheck command in dockerfile but we have 2 different URLs for healthcheck(one for app, other for celery). Also we dont' have to add the healthcheck in docker-compose file for respective containers. Then how can I do it? Can you help me with this. 

Also, To restart brain dead containers, we have to use docker swarm?

Reply:
See container start shell file

Reply:
@iamareebjamal Got it have to access the deployment variable in dockerfile and based on value of it add healthcheck? correct?